build/configs/rtl8730e/loadable_ext_psram/defconfig

```diff
@@ -172,8 +172,8 @@ CONFIG_AMEBASMART_SPI_DMA=y
 # CONFIG_SPI_CS is not set
 CONFIG_AMEBASMART_I2C=y
 CONFIG_AMEBASMART_I2C0=y
-# CONFIG_AMEBASMART_I2C1 is not set
-# CONFIG_AMEBASMART_I2C2 is not set
+CONFIG_AMEBASMART_I2C1=y
+CONFIG_AMEBASMART_I2C2=y
```

Comment:
As far as I know, we use I2C2 only. We don't use I2C1 and I2C0. 
@zhongnuo-tang Even we use I2C2 only, should we enable other config together? I mean in bsp, is there any dependency for theam?